cmd/cloud.go

```diff
@@ -343,20 +344,57 @@ func getCmdCloud(gs *state.GlobalState) *cobra.Command {
 	}
 
 	exampleText := getExampleText(gs, `
-  {{.}} cloud script.js`[1:])
+  # Authenticate with Grafana Cloud k6
+  $ {{.}} cloud login
+
+  # Run a k6 script in the Grafana Cloud k6
+  $ {{.}} cloud run script.js
+
+  # Run a k6 archive in the Grafana Cloud k6
+  $ {{.}} cloud run archive.tar
+
+  # [deprecated] Run a k6 script in the Grafana Cloud k6
+  $ {{.}} cloud script.js
+
+  # [deprecated] Run a k6 archive in the Grafana Cloud k6
+  $ {{.}} cloud archive.tar`[1:])
 
 	cloudCmd := &cobra.Command{
 		Use:   "cloud",
 		Short: "Run a test on the cloud",
-		Long: `Run a test on the cloud.
+		Long: `[deprecation notice]
```

Comment:
Here in for other subcommands. There is a `Deprecated` field of the `cobra.Command`, which probably could serve better for the marking command as deprecated.

Reply:
Oh I didn't know that! Thanks for the pointer, will switch to that indeed 🙇🏻